backend/census_historical_migration/workbooklib/findings.py

```diff
@@ -32,6 +33,30 @@ def xform_sort_compliance_requirement(findings):
         finding.TYPEREQUIREMENT = "".join(sorted(value))
 
 
+def xform_missing_compliance_requirement(findings):
+    """Defaults missing compliance_requirement to GSA_MIGRATION."""
+    change_records = []
+    is_empty_compliance_requirement_found = False
+
+    for finding in findings:
+        compliance_requirement = string_to_string(finding.TYPEREQUIREMENT)
+        if not compliance_requirement:
+            track_transformations(
+                "TYPEREQUIREMENT",
+                finding.TYPEREQUIREMENT,
+                "compliance_requirement",
```

Comment:
There is no compliance_requirement column in dissemination_finding.

Reply:
Fixed